go.mod

```diff
@@ -10,9 +10,18 @@ require (
 	github.com/golang/protobuf v1.5.3
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/stretchr/testify v1.8.4
+	google.golang.org/genproto/googleapis/api v0.0.0-20231009173412-8bfb1ae86b6c
```

Comment:
I assume this is just a side effect of changing version?

Reply:
Yes, the `go mod tidy` did all of these. Which I see happen on most commits as a "side effect".